pkg/collector/corechecks/net/ntp.go

```diff
@@ -171,21 +171,19 @@ func (c *NTPCheck) Run() error {
 
 	clockOffset, err := c.queryOffset()
 	if err != nil {
-		log.Info(err)
-		serviceCheckStatus = metrics.ServiceCheckUnknown
+		log.Error(err)
+		return err
```

Comment:
Should the `ntp.in_sync` service check not be sent at all when there is an error?

Reply:
Service Check is now sent with `UNKNOWN` status when there is an error